scripts/test/SANS/algorithm_detail/q_resolution_calculator_test.py

```diff
@@ -77,7 +77,7 @@ def _provide_mock_state(use_q_resolution, **kwargs):
         mock_state.data.instrument = SANSInstrument.LARMOR
         convert_to_q = MockContainer()
         convert_to_q.use_q_resolution = use_q_resolution
-        for key, value in kwargs.items():
+        for key, value in list(kwargs.items()):
```

Comment:
`2to3` is very cautious. If you had a list in python 2, it will transform into a list in python 3, even if you don't need one. `kwargs.items()` is already an iterable, there is no reason to transform it into a list